model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -290,6 +290,18 @@
     LLMInferenceFramework.TENSORRT_LLM: [],
 }
 
+# This was removed in ba68b8da2f6ef1f6df3f703b35456b72fbe20176. Do we need this if we are not
+# validating max_model_len?
+_MAX_MODEL_LENGTH_CONFIG: Dict[str, Dict[str, Optional[int]]] = {
```

Comment:
let's remove; we don't want to be maintaining a hardcoded list of configs since things change so much. Given that our defaults work out of box, we can assume that our users are educated enough to provide the right model length if they need to deviate from default behavior

Reply:
Fair enough 👍